generate_eval_config_from_checkpoint.py

```diff
@@ -253,43 +253,94 @@ def main(
     if scheduler:
         new_config["scheduler"] = scheduler
 
+    optimizer = OrderedDict()
+    optimizer["name"] = "decoupled_adamw"
+    optimizer["lr"] = 3.0e-5
+    optimizer["betas"] = [0.9, 0.98]
+    optimizer["eps"] = 1.0e-06
+    optimizer["weight_decay"] = 3.0e-6
+    if optimizer:
+        new_config["optimizer"] = optimizer
+
     tasks = OrderedDict()
 
     if not skip_semipro:
         mlmmlu_amateur_semipro = OrderedDict()
         mlmmlu_amateur_semipro["seeds"] = seeds[:4]
         mlmmlu_amateur_semipro["trainer_kwargs"] = {"save_num_checkpoints_to_keep": 0}
+        mlmmlu_amateur_semipro["optimizer"] = OrderedDict([
+            ('name', 'decoupled_adamw'),
+            ('lr', 2.0e-5),
+            ('betas', [0.9, 0.98]),
+            ('eps', 1e-06),
+            ('weight_decay', 5.0e-06),
+        ])
```

Comment:
I'm not sure that adding a second set of hardcoded defaults is the best idea, but I'm not sure there's an easy way to support a dynamic input in typer,

Reply:
One option is to remove the hardcoded optimizer defaults from the `ClassificationJob` classes. I think we want this driven in the config for two reasons: 

1. It's clear to the user how the optimizer is setup without them having to go look in the the classes for each task.
2. It will make logging these things in wandb a bit easier (not 100% on this)